src/custom/utils/rinkeby/constants.ts

```diff
@@ -0,0 +1,26 @@
+import { Token } from '@uniswap/sdk-core'
+import { SupportedChainId as ChainId } from 'constants/chains'
+
+export const DAI_RINKEBY = new Token(ChainId.RINKEBY, '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea', 18, 'DAI', 'DAI')
+
+export const USDT_RINKEBY = new Token(
+  ChainId.RINKEBY,
+  '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa',
```

Comment:
Both addresses are the same

![screenshot_2021-07-29_11-06-24](https://user-images.githubusercontent.com/43217/127543101-e7f84814-ed8b-485c-b677-5fa41cab857f.png)

Reply:
🙈